app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -11,6 +11,10 @@ class OmniauthCallbacksController < Devise::OmniauthCallbacksController
 
     def handle(provider)
       auth_hash = request.env['omniauth.auth']
+      if auth_hash.info.email.nil?
```

Comment:
Is .nil? actually need for some reason? I would expect `if auth_hash.info.email ` to work just fine. Can you please double check?

Reply:
Yes, there is a need to check if email is nil. Instead of the highlighted line, I could use 
"unless auth_hash.info.email"

Reply:
@ViditChitkara yes, let's do that!

Reply:
done